src/core_ocean/shared/mpas_ocn_tracer_short_wave_absorption_jerlov.F

```diff
@@ -193,13 +195,12 @@ subroutine ocn_tracer_short_wave_absorption_jerlov_tend(meshPool, forcingPool, i
       if(config_enable_shortwave_energy_fixer) then
 #ifndef CPRPGI
          !$omp parallel
-         !$omp do schedule(runtime) private(k, layerFraction)
+         !$omp do schedule(runtime) private(k)
 #endif
          do iCell = 1, nCells
-            layerFraction = bottomlayerShortwaveTemperatureFlux(iCell) / (maxLevelCell(iCell) + 1.0E-15_RKIND)
-            do k=1,maxLevelCell(iCell)
-               tend(index_temperature, k, iCell) = tend(index_temperature, k, iCell) + layerFraction
-            end do
+            k = maxLevelCell(iCell)
+            tend(index_temperature, k, iCell) = tend(index_temperature, k, iCell) + &
+                      bottomLayerShortwaveTemperatureFlux(iCell)
```

Comment:
Is it possible that this line could heat a bottom layer at 5000m? Are we sure we won't heat the deep ocean? This could be tested by writing out an instantaneous `bottomLayerShortwaveTemperatureFlux`, make a log color bar, and make sure it is non-zero only at shallow depths - I would expect 100m or less.

Reply:
I have indeed done this, there is no heating below 200 m.  Further down in the file (get_weights function) there is a cap at 200 m that sets weight to zero at 200m capping the effect of this change.  I can also post a figure too if you’d like to see that analysis